watertap/core/util/scaling.py

```diff
@@ -0,0 +1,149 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module contains utility functions for model statistics of WaterTAP models.
+"""
+
+import pyomo.environ as pyo
+from pyomo.environ import check_optimal_termination
```

Comment:
Why import this separately from above? You can just do `pyo.check_optimal_termination`.